pkg/metrics/metrics.go

```diff
@@ -80,6 +80,16 @@ var (
 			"to_version",
 		},
 	)
+
+	RHOAMStatus = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Name: "rhoam_status",
```

Comment:
we're using the name of the cr to reference this query in the alerts. if the name of the cr changes for any reason, this would break. if we expect that not to change we are good but just wanted to call it out.

Reply:
Thanks, as agreed during our call, we don't need to worry about it